sketch/sketch.ts

```diff
@@ -31,19 +40,19 @@ function preload() {
 
 
 function setup() {
-    createCanvas(windowWidth, windowHeight)
+    createCanvas(200, 200)
```

Comment:
its not a major thing but change this back otherwise we will each have to change it back after you merge :)